main.tf

```diff
@@ -40,6 +40,7 @@ resource "aws_cloudformation_stack" "exasol_cluster" {
     Name          = "exasol-${var.cluster_name}-${var.environment}"
     Project       = var.project
     "exa:project" = var.project
+    "exa:project.name" = var.project_name
```

Comment:
Can you align them?
The build also fails because of formatting. Please check that `./scripts/ci.sh` runs successfully in local machine.